clients/cobol-lsp-vscode-extension/src/__tests__/services/copybook/CopybookResolveURITest.spec.ts

```diff
@@ -294,4 +312,34 @@ describe("Prioritize search criteria for copybooks test suite", () => {
     expect(spySearchInWorkspace).toBeCalledTimes(1);
     (globSync as any) = jest.fn().mockReturnValue((x: any) => x);
   });
+  test("Local folder with a relative path given at settings.json, performs search local respective folder", async () => {
+    vscode.workspace.getConfiguration = jest.fn().mockReturnValue({
+      get: jest.fn().mockReturnValue([CPY_FOLDER_NAME]),
+    });
+
+    (globSync as any) = jest.fn().mockReturnValue([copybookName]);
+    const uri: string | undefined = await resolveCopybookHandler(
+      "/storagePath",
+      copybookName,
+      "PRGNAME",
+      "DIALECT",
+    );
+    expect(uri).not.toContain("/storagePath");
+    const fileP = path.join(__dirname, CPY_FOLDER_NAME, copybookName);
+    expect(uri).toEqual("file://" + fileP);
```

Comment:
This fails on windows, becuase fileP is an actual filesystem path.

Reply:
should resolve : 8ca6f339a53f3dfe7e6c969a4ae6e4487daca28a

Reply:
it is removed : 47e087844f7d26bc27e8782dd3412a2815cee89d